appinfo/routes.php

```diff
@@ -32,6 +32,14 @@
 				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleStar',
 				'verb' => 'POST'],
 			[
+				'name' => 'messages#toggleUnseen',
+				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleUnseen',
+				'verb' => 'POST'],
+			[
+				'name' => 'messages#getUnseenCount',
+				'url' => '/accounts/{accountId}/folders/{folderId}/getUnseenCount',
+				'verb' => 'POST'],
```

Comment:
These functions change a state on the IMAP server. Why would that not qualify as a POST?

These two route's functions behave the same at toggleStar, which you wrote (git blame), and I used as references for these routes/functions.